INSTALL.md

```diff
@@ -33,10 +48,20 @@ Now you can start the EtherCAT master:
 ```shell
 $ sudo /etc/init.d/ethercat start
 ```
+it should print
+```shell
+Starting EtherCAT master 1.5.2  done
+```
+
 Make sure your current user is member of `ecusers` and check connected slaves:
 ```shell
+$ sudo chmod 777 /dev/EtherCAT0
```

Comment:
if you are part of `ecusers` you should not need this.

Reply:
okay. removed this line in the latest commit.